src/components/DocumentVIewerSImpleDisplay.tsx

```diff
@@ -0,0 +1,11 @@
+// @ts-ignore
```

Comment:
Please check your spelling in the file name, there are two misplaces capital `I`'s

Reply:
Also does the ts-ignore need to be there?

Reply:
fixed